documentation/src/main/wiki/Assembler.md

```diff
@@ -0,0 +1,137 @@
+#Fragment Assembler
+Fragment Assembler joins all Fragments into the final output. It occurs after Knots processing and
```

Comment:
fixed:
It's executed at the end of all suitable Knots processing, just before generating the response to the page visitor.